### DISCUSSION TITLE
Great Project - Please open up to community to English as well...

### DISCUSSION THREAD

**Discussion by myatix** [camueller/SmartApplianceEnabler#383] [Allgemein]
It would be really great to have this repository also available in English rather than having to use Google Translate.

**Comment by camueller** [comment]
So far the web application (optionally) and the application log is already in English. Only the documentation is still German-only since there aren't many non-german speaking users I'm aware of. If I'm wrong I could make the effort and provide English documentation as well. Where are you from? How did you find this project?

**Comment by myatix** [comment]
My guess is the reason you don't have many English users is because the documentation is in German, this means it's almost impossible for English speakers to find this solution without you pointing it out to them manually in other forums! ;)
If it was in English people might have a chance of discovering this exists and then you will also get more people in the community and potentially more testers. :)

**Reply by camueller** [reply]
You did not answer my questions 🙃

From time to time I'm checking for English speaking forums and discussion groups of Sunny Home Manager users in order to rectify the effort of translation but so far I have found nothing compared to German speaking photovoltaikforum.com .

Perhaps you can point me to some you know...

**Comment by myatix** [comment]
I am English but live in Denmark...
You replied to a question I left on the DIY Solar Power Forum with regards to Sunny Home Manager IFTTT integration. :)

[http](https://diysolarforum.com/threads/sma-home-manager-2-0-and-ifttt.47219/)

other forums I am a member of that are relevant are buildhub.org.uk, but you need to be a resident of the UK to be a member. 

If you want me to promote your solution I can but you need to put the site in English first, otherwise users can't really participate. :)

**Reply by camueller** [reply]
Okay then, I have some time over Christmas I can use to translate everything. In the meantime you will have to use Google Translate. If you have questions I can also help.

**Comment by camueller** [comment]
@myatix I started the translation on a separate branch:

https://github.com/camueller/SmartApplianceEnabler/blob/english/README_EN.md

The main page is translated except for graphics and videos.

I really would appreciate if you could have loook the translated pages from time to time over the next few weeks and provide feedback (wording, ...)

**Comment by myatix** [comment]
Hi Alex,

I have read through the English page and the translation is looking great.

Nice work! 😀

Best regards

Alan

**Comment by camueller** [comment]
All pages are translated (with help of Google translate). There should be plenty for you to read :-)

The images embedded still need to be recreated using english frontend settings.

**Comment by camueller** [comment]
Most of the images have be captured again - for german and english front end.

I have to read the english translation fully one time. Afterwards I will merge it to the master branch.

**Comment by camueller** [comment]
Meanwhile I have merged the english documentation to the main branch.

At the top of the home page there is a link leading to it.
